tests/rptest/tests/cluster_config_test.py

```diff
@@ -1226,6 +1226,31 @@ def yamlize(input) -> str:
                     f"Config setting {key}={strval} should have been rejected")
                 assert False
 
+        # Check that the `get` command hits proper validation paths
+        unknown_examples = [
+            "panda_size", "panda_retention_ms", "panda_mutation_rate"
+        ]
+
+        for key in unknown_examples:
+            try:
+                self.rpk.cluster_config_get(key)
+            except RpkException as e:
+                pass
+            else:
+                self.logger.error(
+                    f"Config getting {key} should have been rejected")
+                assert False
+
+        for key in unknown_examples:
+            try:
+                self.admin.get_cluster_config(key=key)
+            except requests.exceptions.HTTPError as e:
+                assert e.response.status_code == 400
+            else:
+                self.logger.error(
+                    f"Config getting {key} should have been rejected")
+                assert False
```

Comment:
nitpick: probably prefer expect_exception anyway, but I find an annotated assert a bit easier to read off the stack trace than a log line and and a bare assert. as in:

```suggestion
                assert False, f"Config getting {key} should have been rejected"
```